index.html

```diff
@@ -41,60 +35,168 @@ <h2 class="primary_color">I'm a software developer</h2>
+          <img src="images/Snapshoot Portfolio-2.png" alt="Tonic Website Snapshoot" />
+        </a>
+        <h3 class="tertiary_color">Tonic</h3>
+        <ul class="d-flex justify-content-between">
+          <li class="li_works tertiary_color">CANOPY</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+
+      <article class="grid_5x1 border">
+        <a href="#">
+          <img
+            src="images/Snapshoot Portfolio-3.png"
+            alt="Multi-Post Stories Gain+Glory Snapshoot"
+          />
+        </a>
+        <h3 class="tertiary_color">Multi-Post Stories</h3>
+        <ul class="d-flex justify-content-center">
+          <li class="li_works tertiary_color">Ruby on Rails</li>
+          <li class="li_works tertiary_color">css</li>
+          <li class="li_works tertiary_color">Javascript</li>
+          <li class="li_works tertiary_color">html</li>
+        </ul>
+        <a class="btn1" href="#">See Projects</a>
+      </article>
+    </section>
+
+    <!-- About me Section -->
+
+    <section class="d-flex flex-column align-items-center">
+      <article class="d-flex flex-column align-items-center about_me">
+        <h2 class="pt-1 secondary_color">About Me</h2>
+        <p class="pt-1 pb-1 p_about_me tertiary_color">
+          I can help you build a product, feature or website. Look through some of my work and
+          experience! If you like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <a class="btn1 mb-1" href="#">Get My Resume</a>
+      </article>
+
+      <ul class="d-flex flex-column">
+        <li>
+          <article class="d-flex flex-column align-items-center border">
+            <img class="pt-4 pb-1" src="images/Frame.png" alt="" />
+            <h3 class="pu-5 secondary_color">Languages</h3>
+            <ul class="d-flex justify-content-center flex-wrap ul_about_me">
+              <li class="li_works primary_color">Javascript</li>
+              <li class="li_works primary_color">Ruby</li>
+              <li class="li_works primary_color">Html</li>
+              <li class="li_works primary_color">Css</li>
+            </ul>
+          </article>
+        </li>
+
+        <li>
+          <article class="d-flex flex-column align-items-center border">
+            <img class="pt-4 pb-1" src="images/Group.png" alt="" />
+            <h3 class="pu-5 secondary_color">Frameworks</h3>
+            <ul class="d-flex justify-content-center flex-wrap ul_about_me">
+              <li class="li_works primary_color">Bootstrap</li>
+              <li class="li_works primary_color">Ruby on Rails</li>
+              <li class="li_works primary_color">RSpec</li>
+              <li class="li_works primary_color">Capybara</li>
+              <li class="li_works primary_color">Selenium</li>
```

Comment:
- [ ] :recycle: In my opinion we should try again to implement again the suggestion of the previous reviewer, by make this section look more like the design provided in the task:

  - Your design:
![Your2](https://user-images.githubusercontent.com/97834160/205410650-f5d90a50-1b69-4cbe-afa2-24c430d3c993.png)

  - Figma design:

![fig2](https://user-images.githubusercontent.com/97834160/205410817-86e3c70e-3877-4be2-935d-eb3b2e4fae53.png)